maturity-matrix.yaml

```diff
@@ -47,9 +47,11 @@ productionMaturityLevels:
   # typical production use cases.
   - stable
 
-# All OpenTelemetry APIs follow semver conventions (i.e.,
-# backwards-incompatibly changes should bump the major version number).
+# All OpenTelemetry APIs follow semver conventions (i.e., after v1.x,
+# backwards-incompatible changes should bump the major version number).
 apiMaturityLevels:
+  # "unimplemented" APIs do not exist yet for the component/language.
```

Comment:
@lizthegrey WDYT about (also) adding something like "unplanned" or "undesired" or "notApplicable"? E.g., auto-instrumentation in some future `otel-c99` support.

Reply:
agree we should introduce n/a.

Reply:
done